sql/core/src/test/resources/tpcds-query-results/v1_4/q1.sql.out

```diff
@@ -3,103 +3,103 @@
 -- !query schema
 struct<c_customer_id:string>
 -- !query output
-AAAAAAAAAAABBAAA
-AAAAAAAAAAADBAAA
-AAAAAAAAAAADBAAA
-AAAAAAAAAAAKAAAA
-AAAAAAAAAABDAAAA
-AAAAAAAAAABHBAAA
-AAAAAAAAAABLAAAA
-AAAAAAAAAABMAAAA
-AAAAAAAAAACHAAAA
-AAAAAAAAAACMAAAA
-AAAAAAAAAADDAAAA
-AAAAAAAAAADGAAAA
-AAAAAAAAAADGBAAA
-AAAAAAAAAADGBAAA
+AAAAAAAAAAABAAAA
```

Comment:
This PR updated the golden files because datagen used to generate `maropu/spark-tpcds-sf-1` was older than the `databricks/tpcds-kit` one.

Reply:
In this case, it seems that we had better split this PR into two. WDYT, @maropu ?

Reply:
yea, sgtm. We can split it by checking out my repo instead: https://github.com/maropu/spark-tpcds-datagen/tree/master/thirdparty/tpcds-kit/tools

Reply:
I'll do it later, too.

Reply:
okay, I've fixed the code so that it generate the same TPC-DS data with https://github.com/maropu/spark-tpcds-sf-1.